src/components/dashboard/Dashboard.jsx

```diff
@@ -32,11 +36,23 @@ export default function Dashboard() {
     }
   }, []);
 
+  useEffect(() => {
+    pushUserCustomerAttributes(enterpriseConfig);
+  }, [enterpriseConfig]);
```

Comment:
I think we'd may want to push user/customer attributes in a more global location in the component tree than the Dashboard page specifically so that we are doing it regardless of the user's entry point into the authenticated application.

Reply:
Maybe in `EnterprisePage`?  Is there a component higher up in the tree that has `enterpriseConfig` context?